tools/cellpose/cp_segmentation.py

```diff
@@ -6,9 +6,20 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import skimage.io
+import torch
 from cellpose import models, plot, transforms
 
 
+# Make sure, that MKL_NUM_THREADS is set to 1, to ensure reproducibility:
+# https://forum.image.sc/t/reproducibility-how-we-spent-years-building-containers-and-then-mkl-decided-to-screw-our-results/109599
+assert str(os.environ['MKL_NUM_THREADS']) == '1'
```

Comment:
instead of asserting, could we also set this variable here again?

Reply:
The question is, at what point MKL is loaded. My guess is that this happens when NumPy is loaded. I have made some adjustments based on your suggestion accordingly in 66fb7256c204e93e2e502d9b721a668178b6e4e2.